### ISSUE TITLE
Frontline between two blue airfields

### ISSUE THREAD

**Issue by alexkrebiehl** [dcs-liberation/dcs_liberation#1068]
**Describe the bug**
A front line exists between two airfields that I own (and units are populated there)

**Version information (please complete the following information):**
 - DCS Liberation v2.5
 
[front line bug.zip](https://github.com/dcs-liberation/dcs_liberation/files/6446258/front.line.bug.zip)

**Comment by DanAlbert** [comment]
Haven't looked, but I suspect were going to need to know how you got into that situation. Have never seen this before.

**Comment by alexkrebiehl** [comment]
Not entirely sure - is turn history saved anywhere? After I captured Mozdok, the frontline reset back to Beslan and started moving towards Mozdok again. I thought it would go away after a turn, but it didn't.

**Comment by DanAlbert** [comment]
```
2021-05-18 19:37:54,964 :: ERROR :: Invalid Save game
Traceback (most recent call last):
  File "C:\Users\dan\src\dcs_liberation\game\persistency.py", line 38, in load_game
    save = pickle.load(f)
AttributeError: Can't get attribute 'FactoryGroundObject' on <module 'game.theater.theatergroundobject' from 'C:\\Users\\dan\\src\\dcs_liberation\\game\\theater\\theatergroundobject.py'>
```

This doesn't appear to be a save from 2.5. Without knowing the exact build of the dev branch this came from I can't debug anything.

Have you seen this problem recur? Since it's from the dev branch there's always a decent chance that it was a temporary breakage.

Closing on the assumption that this is either no longer a bug, or at the very least not one we can investigate. Will reopen if we get the data we need.

**Comment by DanAlbert** [comment]
That's my mistake. I'd saved your files to the wrong directory but happened to already have a caucasus.liberation in the correct directory :)

**Comment by DanAlbert** [comment]
![image](https://user-images.githubusercontent.com/315852/118751629-7f54fe80-b816-11eb-9020-e9f4fe3c0b4a.png)

According to the game that front line is between Beslan and Nalchik, not Beslan and Mozdok, but it's definitely placed on the Beslan/Mozdok line. Can be easily repro'd and that campaign (Northern Caucasus) by cheating to capture Beslan and then Mozdok.

This campaign is the old format, so I suspect this is a bug in that. The old format isn't available in 3.0 any more, so closing.
